server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSOptionsCheckBaseUtility.java

```diff
@@ -170,6 +165,42 @@ private void throwException(
     }
   }
 
+  private void checkDuplicateEntries(
+      ParserRuleContext ctx,
+      Map<Integer, TerminalNode> entries,
+      Map<String, ErrorSeverity> specialSeverities) {
+    if (ctx.getChildCount() != 0) {
+      for (ParseTree entry : ctx.children) {
+        if (entry.getChildCount() == 0) {
+          TerminalNode current = (TerminalNode) entry;
```

Comment:
I don't think this is a valid test. You can have empty RuleContexts that are not TerminalNodes.